.github/workflows/release-docker-image.yml

```diff
@@ -98,11 +106,7 @@ jobs:
         with:
           result-encoding: string
           script: |
-            return (
-              context.eventName === 'workflow_dispatch'
-                ? '${{ github.event.inputs.tag }}'
-                : context.payload.release.tag_name
-            );
+            return ( "${{ inputs.tag }}" ||  context.payload.release.tag_name );
```

Comment:
why do we still need this if `inputs.tag` default is `github.event.inputs.tag || github.event.release.tag_name`?